cli/.js/src/main/scala/aqua/Prelude.scala

```diff
@@ -0,0 +1,38 @@
+package aqua
+
+import aqua.LogLevelTransformer
+import aqua.js.{Meta, Module}
+import fs2.io.file.Path
+import scribe.Logging
+
+import scala.util.Try
+
+/**
+ * @param pathList list of paths where imports will be searched
+ */
+case class Prelude(pathList: List[Path])
+
+// JS-specific functions
+object Prelude extends Logging {
+
+  def apply(): Prelude = {
```

Comment:
Please not `apply`, smth else, or we will die trying to refactor it in the future